<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveWindow</class>
 <widget class="QWidget" name="SaveWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="FilenameInput">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>171</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="FilenameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя файла:</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="SaveButton">
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ExitButton">
      <property name="text">
       <string>Выйти</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
